kuwala/core/canvas/src/components/Common/Block/index.js

```diff
@@ -0,0 +1,118 @@
+import {useStoreActions} from "easy-peasy";
+import Classes from "../../Nodes/TransformationBlock/TransformationBlockStyle";
+import Icon from "../Icon";
+import Button from "../Button";
+import {Handle} from "react-flow-renderer";
+import {COLOR_MAP, KUWALA_GRAY, KUWALA_PURPLE} from "../../../constants/styling";
+import React from "react";
+
+const Block = ({
+        blockColor='kuwalaGreen',
+        title='Title',
```

Comment:
Shouldn't have a default value as we want to make sure it is always passed down to the component.